biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -246,6 +246,8 @@ BAM alerts require BAM tools to be enabled.
 ### Configure the REST APIs
 >[NOTE]
 >Start from BizTalk Server 2016 feature pack, you can use REST APIs to manage the different artifacts in your BizTalk Server enviroments and operational data flowing through. 
```

Comment:
Since this is the config guide and config module is available starting 2020, I don't think you should mention the note about 2016 feature pack. Those details are already there in FP docs